### PR TITLE
Add Model F-inspired keymap to YMD[K]96 keyboard

### PR DESCRIPTION
**Pull request by elvis-epx**
## Description

Add a keymap to YMD[K]96 keyboard. It is inspired on Model F, with arrow block activated by Num Lock. To be usable in Macs, Num Lock key toggles a layer instead of being sent to the computer. 

## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by this PR

* 

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ymd96/keymaps/epx/keymap.c [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public LicensezZZ
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "ymd96.h"
```
Comment (drashna) [inline keyboards/ymd96/keymaps/epx/keymap.c]: Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```

File: keyboards/ymd96/keymaps/epx/keymap.c [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+#include "action_layer.h"
```
Comment (drashna) [inline keyboards/ymd96/keymaps/epx/keymap.c]: Could you remove all of these additional includes?  

```suggestion
```

They get included automatically, at various points in the built process, so they are redundant and not necessary here.

File: keyboards/ymd96/keymaps/epx/keymap.c [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+#include "rgblight.h"
```
Comment (drashna) [inline keyboards/ymd96/keymaps/epx/keymap.c]: Could you remove all of these additional includes?  

```suggestion
```

They get included automatically, at various points in the built process, so they are redundant and not necessary here.

File: keyboards/ymd96/keymaps/epx/keymap.c [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+#define _______ KC_TRNS
```
Comment (drashna) [inline keyboards/ymd96/keymaps/epx/keymap.c]: This define isn't necessary, as it is actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

**Comment by elvis-epx** [comment]
Made the changes, tested on my kbd and squashed the commits into one.

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**